### PR TITLE
Hls 2168 4

### PR DESCRIPTION
**Pull request by LironKS**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -30,19 +30,14 @@ jobs:
   release:
     needs:
       - assemble
-      - test
+#      - test
```
Comment (nozik) [inline .github/workflows/release.yml]: Why not delete all the commented out stuff?

File: version.gradle.kts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-val stableVersion = "1.16.0-SNAPSHOT"
-val alphaVersion = "1.16.0-alpha-SNAPSHOT"
+val stableVersion = "1.17.0-SNAPSHOT"
```
Comment (nozik) [inline version.gradle.kts:1]: This is where we control the version of the release?

**Review by nozik — CHANGES_REQUESTED** [review]

File: version.gradle.kts (hunk 2 above)
Comment (LironKS) [reply version.gradle.kts:1]: yes

File: .github/workflows/release.yml (hunk 1 above)
Comment (LironKS) [reply .github/workflows/release.yml]: it won't be comment out in the release job it's only for me to test the job now

**Review by nozik — APPROVED** [review]

**Pull request closed without merging**